okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -276,6 +279,9 @@ public void onResponse(Call call, Response response) throws IOException {
     }
 
     private boolean shouldRetry(IOException exception, Optional<Duration> backoff) {
+        if (retryOnSocketException == ClientConfiguration.RetryOnSocketException.DANGEROUS_DISABLED) {
+            return false;
```

Comment:
it seems like this would be equivalent to setting retryOnTimeout to DISABLED and setting backoff to empty, why was that configuration insufficient?

Reply:
we still want to backoff on things like 429s and 503s